src/domains/qi-common/02_security.tf

```
@@ -41,7 +41,7 @@ resource "azurerm_key_vault_access_policy" "adgroup_developers_policy" {
   tenant_id = data.azurerm_client_config.current.tenant_id
   object_id = data.azuread_group.adgroup_developers.object_id
 
-  key_permissions     = ["Get", "List", "Update", "Create", "Import", "Delete", ]
+  key_permissions     = ["Get", "List", "Update", "Create", "Import", "Delete", "Encrypt", "Decrypt"]
```

@pasqualespica here we add permissions also for Decrypt but maybe it should be lowered to Encrypt only. I take those permissions from [here](https://github.com/pagopa/pagopa-infra/blob/3dfcd939c2927a8e58cc0851cc79906fb0b02b74/src/domains/receipts-secrets/02_security.tf#L34-L49)
```suggestion
  key_permissions     = ["Get", "List", "Update", "Create", "Import", "Delete", "Encrypt"]
```
What do you think?